src/CoreWCF.WebHttp/src/CoreWCF/Channels/WebMessageEncoderFactory.cs

```diff
@@ -0,0 +1,436 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using CoreWCF.Diagnostics;
+using CoreWCF.Runtime;
+
+namespace CoreWCF.Channels
+{
+    public class WebMessageEncoderFactory : MessageEncoderFactory
+    {
+        WebMessageEncoder _messageEncoder;
+
```

Comment:
Missing accessibility modifier

Reply:
This is addressed in 9d3dc81482713d2633975f33deda359ce416f63d.